MANIFEST.in

```diff
@@ -11,6 +11,12 @@ prune answer-store
 recursive-include yt *.py *.pyx *.pxi *.pxd *.h *.hpp README* *.txt LICENSE* *.cu
 recursive-include doc *.rst *.txt *.py *.ipynb *.png *.jpg *.css *.html
 recursive-include doc *.h *.c *.sh *.svgz *.pdf *.svg *.pyx
+recursive-exclude yt *.cpp *.c
+include yt/utilities/lib/fixed_interpolator.cpp
+include yt/utilities/lib/pixelization_constants.cpp
+include yt/utilities/lib/origami_tags.c
+include yt/utilities/lib/tsearch.c
```

Comment:
Is there a reason not to include all files found with
```bash
$ git ls-files | grep -E '\.[hpp|cpp|c]$'
doc/source/analyzing/_static/axes.c
doc/source/analyzing/_static/axes.h
yt/frontends/artio/artio_headers/artio.c
yt/frontends/artio/artio_headers/artio.h
yt/frontends/artio/artio_headers/artio_endian.c
yt/frontends/artio/artio_headers/artio_endian.h
yt/frontends/artio/artio_headers/artio_file.c
yt/frontends/artio/artio_headers/artio_grid.c
yt/frontends/artio/artio_headers/artio_internal.h
yt/frontends/artio/artio_headers/artio_mpi.c
yt/frontends/artio/artio_headers/artio_mpi.h
yt/frontends/artio/artio_headers/artio_parameter.c
yt/frontends/artio/artio_headers/artio_particle.c
yt/frontends/artio/artio_headers/artio_posix.c
yt/frontends/artio/artio_headers/artio_selector.c
yt/frontends/artio/artio_headers/artio_sfc.c
yt/frontends/artio/artio_headers/cosmology.c
yt/frontends/artio/artio_headers/cosmology.h
yt/geometry/vectorized_ops.h
yt/utilities/lib/cykdtree/windows/stdint.h
yt/utilities/lib/endian_swap.h
yt/utilities/lib/marching_cubes.h
yt/utilities/lib/mesh_triangulation.h
yt/utilities/lib/origami_tags.c
yt/utilities/lib/origami_tags.h
yt/utilities/lib/platform_dep.h
yt/utilities/lib/tsearch.c
yt/utilities/lib/tsearch.h
```

Reply:
I think we should avoid excluding anything that we bundle, and instead focus on *just* excluding the Cython generated sources.  Certainly for sdist these are all necessary.

Reply:
Headers are already included by previous instructions (and should be safe since they're never generated as far as I know). I only included the bare minimum set of files that made `python -m build` run without errors, and  overlooked the rest. Now I'm actually not sure why this seems to work, I guess we are not properly testing that sdists we produce are functional.

Reply:
I don't think we need the .c and .cpp files since the .so files are included in the wheel. However, in the source, we definitely need them (have you tried compiling from the mere .tar.gz file?).

Reply:
I haven't. I'm thinking we should enable some for of testing in the wheel building workflow first.
Let me draft this for now.

Reply:
Ok now we got *some* form of check rolling, and I addressed this remark (but the `grep` invoke appeared to be slightly broken so I commited a fixed version of it together with the resulting list)